packetbeat/config/config.go

```diff
@@ -5,19 +5,19 @@ import (
 
 	"github.com/elastic/beats/libbeat/common"
 	"github.com/elastic/beats/libbeat/common/droppriv"
-	"github.com/elastic/beats/libbeat/logp"
-	"github.com/elastic/beats/libbeat/publisher"
 	"github.com/elastic/beats/packetbeat/procs"
 )
 
 type Config struct {
+	Packetbeat PacketbeatConfig
```

Comment:
alternatively this could have been written like:

```
type Config struct {
  Interfaces InterfacesConfig `config:"packetbeat.interfaces"`
  Flows      *Flows           `config:"packetbeat.flows"`
  ...
}
```